### PR TITLE
CAS-6 Feature/parse markdown

### PR DESCRIPTION
**Pull request by andriizhumela**
Add checking with regular for message list holder.

### CODE REVIEW

File: sample/src/main/java/io/getstream/chat/example/MainActivity.java [hunk 1]
```diff
@@ -206,7 +207,10 @@ void createNewChannel(String channelName) {
         extraData.put("members", members);
 
         String channelId = channelName.replaceAll(" ", "-").toLowerCase();
+
         Channel channel = new Channel(client, ModelType.channel_messaging, channelId, extraData);
+        Config conf = new Config();
```
Comment (adrian09h) [inline sample/src/main/java/io/getstream/chat/example/MainActivity.java]: What is this for?

**Comment by adrian09h** [comment]
Can you please add test for `isHasMarkdown`?

File: library/src/main/java/com/getstream/sdk/chat/utils/StringUtility.java [hunk 2]
```diff
@@ -40,6 +43,12 @@ public static boolean isEmoji(String message) {
                 "[\u231A\u231B\u2328\u23CF\u23E9-\u23F3\u23F8-\u23FA]\uFE0F?)+");
     }
 
+    public static boolean isHasMarkdown(String message) {
+        String pattenr = "[*@_~#{\\[\\]\"'|>]";
```
Comment (tbarbugli) [inline library/src/main/java/com/getstream/sdk/chat/utils/StringUtility.java]: typo: pattenr -> pattern

File: library/src/main/java/com/getstream/sdk/chat/utils/StringUtility.java [hunk 3]
```diff
@@ -40,6 +43,12 @@ public static boolean isEmoji(String message) {
                 "[\u231A\u231B\u2328\u23CF\u23E9-\u23F3\u23F8-\u23FA]\uFE0F?)+");
     }
 
+    public static boolean isHasMarkdown(String message) {
```
Comment (tbarbugli) [inline library/src/main/java/com/getstream/sdk/chat/utils/StringUtility.java]: better to call this `containsMarkdown`

File: library/src/main/java/com/getstream/sdk/chat/utils/StringUtility.java [hunk 4]
```diff
@@ -40,6 +43,12 @@ public static boolean isEmoji(String message) {
...
+        String pattenr = "[*@_~#{\\[\\]\"'|>]";
+        boolean isHasMarkdown = Pattern.compile(pattenr).matcher(message).find();
```
Comment (tbarbugli) [inline library/src/main/java/com/getstream/sdk/chat/utils/StringUtility.java]: this method is going to be called *a lot*, what about we move the `Pattern.compile` to a private attr?

File: sample/src/main/java/io/getstream/chat/example/MainActivity.java (hunk 1 above)
Comment (andriizhumela) [reply sample/src/main/java/io/getstream/chat/example/MainActivity.java]: Removed. (i forgot what i planned to do with this)

**Comment by andriizhumela** [comment]
> Can you please add test for `isHasMarkdown`?
> Please add all possible cases for Morkdown text to the test.

@adrian09h I used `Pattern.compile` from android skd. As far as i kwon we can check it only with instrumental tests(not Unit test). Do we have instrumental tests?

**Pull request merged**